doc/website/scripts/generate_module_pages.py

```
@@ -0,0 +1,519 @@
+    5: "#1a5c2e",
+}
+
+
+# ---------------------------------------------------------------------------
+# Repository root resolution
+# ---------------------------------------------------------------------------
+
+
+def get_holohub_root() -> Path:
+    result = subprocess.run(
+        ["git", "rev-parse", "--show-toplevel"], capture_output=True, text=True, check=True
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)